pages/news/2020-XX-XX-concrete-action.html

```diff
@@ -0,0 +1,63 @@
+title: Concrete Action: Taking a Stand with Terminology and Recruitment
+author: Mixxx, April
+date: 2020-06-08 01:00:00
+
+{% extends "post.html" %}
+
+{% block post %}
+
+{% load markup %}
+{% filter markdown:"extra" %}
+
+# &#35;BlackLivesMatter
+
+Alongside the protesters in the US and around the world, the Mixxx team unequivocally renounces police brutality.
```

Comment:
```suggestion
Alongside the protesters in the US and around the world, the Mixxx team unequivocally despises police brutality.
```

I am sure it makes sense in the US but this sentence should be international unambiguous.
According to DeepL this translates into "verzichten auf Polizeibrutalität", which sound as If we have committed brutality in the past. I think It should be "verachtet Polizeibrutalität" which is "despises police brutality"

Reply:
How about "condemn"?